.github/workflows/ci.yml

```diff
@@ -202,6 +202,34 @@ jobs:
 
             exit 1
           fi
+
+          kubectl wait --for=condition=Ready=true node/kind-worker --timeout=60s
```

Comment:
Suggestion: create a `for` loop to iterate over the three nodes.

Reply:
Done